### DISCUSSION TITLE
要是可以出一个封装的思路教程就好了

### DISCUSSION THREAD

**Discussion by okay-xiaomo** [zouyaoji/vue-cesium#431] [General]
要是可以出一个封装的思路教程就好了，源码看起来比较复杂，每个功能都有好多的引用，因为我们自己的项目一直还是在用vue2所以 想学习一下思路，把在Vue3的这个测量的工具全部移植到vue2中 😁

**Comment by zouyaoji** [comment]
建议是用vue3版本哈。
